packages/bootleg-tokens/contracts/ISharedRoyaltyToken.sol

```diff
@@ -0,0 +1,64 @@
+pragma solidity ^0.5.3;
+
+import "openzeppelin-solidity/contracts/token/ERC721/IERC721.sol";
+
+/**
+ * @title Shared Royalty Non-Fungible Token Standard basic interface
+ */
+contract ISharedRoyaltyToken is IERC721 {
+
+  /**
+   * @notice Gets the franchisor balance of the specified address
+   *
+   * @param _franchisor address to query the balance of
+   * @return uint256 representing the amount franchised by the passed address
+   */
+  function franchisorBalanceOf(address _franchisor) public view returns (uint256 balance);
+
+  /**
+   * @notice Withdraws payment accumulated from transfer of a given token from
+   *  the last withdrawn payment up to a _count
+   *
+   * @param _tokenId The identifier for an NFT
+   * @param _count The number of payments to traverse
```

Comment:
Could you explain a little more about the "payments to traverse" part. I'm not sure I get it.